frc46_token/src/token/types.rs

```diff
@@ -118,7 +118,7 @@ pub struct MintReturn {
     pub balance: TokenAmount,
     /// The new total supply.
     pub supply: TokenAmount,
-    /// (Optional) data returned from receiver hook
+    /// (Optional) CBOR-encoded data returned from receiver hook
```

Comment:
I don't think anything here forces this to be CBOR, and this comment is likely to rot. cc @Stebalien

Reply:
We should probably just switch this over to `Option<IpldBlock>` (or actually check the codec).

Reply:
https://github.com/helix-onchain/filecoin/issues/178